backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs

```diff
@@ -156,6 +156,11 @@ public async Task<AvailabilitySlotDto> GetAvailabilitySlotById(long id)
                 throw new KeyNotFoundException("Availability slot doesn't exist");
             }
 
+            if (!availabilitySlot.SlotMembers.Any())
+            {
+                availabilitySlot.SlotMembers.Add(new SlotMember { Schedule = new Schedule() });
```

Comment:
This looks like a workaround than solutions

Reply:
Maybe. But in mapper profile we have rule that can broke endpoint if slot member collection will be empty. By this changes we don't affect anything and can return response

Reply:
you need to find another way to fix it

Reply:
You use hardcoded Ids this wont fix all developers DBs

Reply:
As I Understand, I need to solve this issue on our back-end side (without sql script, if we speak about all developers' DBs) ?

Reply:
you can do it in the backend and in the DBs
Two cases are good and can be implemented